Automattic/wp-calypso
# Add unified design picker

#### Proposed Changes
Add a unified design picker as outlined in https://github.com/Automattic/wp-calypso/issues/64998
We've implemented a basic version, basically combining existing functionality into a unified experience rather than trying to implement every new feature in the mockups.
Built to use the new api method `/starter-designs` implemented in D84155-code
<img width="1728" alt="Screen Shot 2022-07-20 at 2 48 21 pm" src="https://user-images.githubusercontent.com/22446385/179899235-17da2a27-2722-4eb2-801b-380c0b62d0fc.png">

#### Testing instructions
Apply D84155-code to your sandbox.
Ensure `signup/design-picker-unified` flag is enabled.
This is quite a substantial feature but it should basically work the same as the existing static design picker.
Testing list in this comment https://github.com/Automattic/wp-calypso/issues/64998#issuecomment-1182724200
